GPU/Common/TextureCacheCommon.cpp

```diff
@@ -1355,7 +1355,8 @@ void TextureCacheCommon::LoadClut(u32 clutAddr, u32 loadBytes) {
 				// the framebuffer with the smallest offset. This is yet another framebuffer matching
 				// loop with its own rules, eventually we'll probably want to do something
 				// more systematic.
-				if (matchRange && !inMargin && offset < (int)clutRenderOffset_) {
+				bool okAge = !PSP_CoreParameter().compat.flags().LoadCLUTFromCurrentFrameOnly || framebuffer->last_frame_used == gpuStats.numFlips;
```

Comment:
This should probably at least account for last_frame_render if it becomes the default?

By the way, this is already in an `if (matchRange) {`.  I could see a game rendering a clut and then reading from it every frame, how old is the buffer in Syphon Filter?  Heurstics like this will probably just get changed later when you figure "it's probably not an issue anymore" unless they have stronger logic to them.

-[Unknown]